primitive-types/impls/serde/src/serialize.rs

```diff
@@ -75,6 +84,65 @@ impl<'a> fmt::Display for ExpectedLen<'a> {
 	}
 }
 
+/// Deserialize into vector of bytes.  This DOES NOT guard against DoS attacks!
```

Comment:
Can you elaborate what you mean by: "This DOES NOT guard against DoS attacks!"?
We actually use that in public RPCs, so it's easily exploitable if you know a vulnerability here.

Do you just mean an allocation of super long string that is not a valid hex?

Reply:
I wonder the same

Reply:
@tomusdrw Correct ― this will allocate an arbitrarily long string.